### PR TITLE
Add drops adapter

### PR DESCRIPTION
**Pull request by freelife2010**
##### Twitter Link:
https://twitter.com/dropsnft

##### Website Link:
https://drops.co

##### Logo (High resolution, preferably in .svg and .png, for application on both white and black backgrounds. Will be shown with rounded borders):

logo
logo_white

##### Current TVL:
7.7M

##### Chain:
Ethereum

##### Short Description (to be shown on DefiLlama):

Get more value from your NFT & DeFi assets
##### Token address and ticker if any:
DOP: 0x6bB61215298F296C55b19Ad842D3Df69021DA2ef

##### Category (Yield/Dexes/Lending/Minting/Assets/Insurance/Options/Indexes/Staking) *Please choose only one:

Lending
##### forkedFrom (Does your project originate from another project):

Compound
##### methodology (what is being counted as tvl, how is tvl being calculated):

TVL is comprised of tokens deposited to the protocol as collateral, similar to Compound Finance and other lending protocols the borrowed tokens are not counted as TVL.

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -1,189 +1,219 @@
...
 
 async function getLatestBlockRetry(chain) {
-    for (let i = 0; i < 5; i++) {
-        try {
-            return await getLatestBlock(chain);
-        } catch (e) {
-            throw new Error(`Couln't get block heights for chain "${chain}"`, e)
-        }
-    }
+  // for (let i = 0; i < 5; i++) {
```
Comment (0xngmi) [inline test.js:13]: why use a fixed block?

Comment (freelife2010) [reply test.js:13]: When I try to run test.js, I got error "Couln't get block heights for chain"
so I have used fixed block for testing

Comment (freelife2010) [reply test.js]: I have reverted test.js code

**Comment by waynebruce0x** [comment]
Can the staking (LP + NDR) be seperated out into a seperate 'staking' export? Also, for money markets we could TVL as supply minus borrow (to prevent double counting recursive supply). Is it possible to count the tokens locked in your contracts? or subtract total borrows?

**Comment by freelife2010** [comment]
we consider TVL as Totalsupply from api(https://drops.co/status) + Total Staking USD

**Comment by waynebruce0x** [comment]
At defillama we try to keep a consistent methodology across all projects, so for all money markets, Aave etc, we dont count include borrowed assets. We also allow users to choose whether they include staking as tvl - this means it needs to be exported separately to core tvl.

**Comment by freelife2010** [comment]
> 

I have updated pr

**Pull request merged**